sdk/sdkconnect/actions.go

```diff
@@ -0,0 +1,70 @@
+// Wrapper for ActionServiceClient (generated code) DO NOT EDIT
```

Comment:
Add this directory as `linguist-generated` to `./.gitattributes` so they collapse by default in the PR review page